packages/email/email.js

```diff
@@ -104,12 +107,15 @@ const knownHostsTransport = function (settings = undefined, url = undefined) {
     },
   });
 
+  if (options?.encryptionKeys || options?.shouldSign) {
```

Comment:
Maybe I miss something, but do we really need to have two configuration options to enable sign?

Reply:
A user could provide the `encryptionKeys` yet wouldn't want to encrypt things yet and he may also provide `shouldSign` but without proper `encryptionKeys` it's meaningless. So, both values must exists in order to be able to carry the encryption correctly.

Reply:
It looks like `shouldSign` in `nodemailer-openpgp` defaults to true and is only explicitly needed if you don't want to sign for some reason. https://github.com/nodemailer/nodemailer-openpgp/blob/1ff06189d0c2e021de09e15bc55bfcc8a30e2a51/lib/nodemailer-openpgp.js#L45

Reply:
Ok, I see then these parameters are just coming from `nodemailer-openpgp` and it will be by default `true` unless passed the contrary.

Thanks both good to know this is just perserving same config options than the original package integrated.